alica_ros_turtlesim/alica_ros_turtlesim.rosinstall

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
This file does not serve any purpose here

Reply:
Currently I wrote following in README
`mkdir -p catkin_ws/src
cd catkin_ws/src
git clone https://github.com/rapyuta-robotics/alica-supplementary.git
cd ..
wstool init
wstool merge src/alica-supplementary/alica_ros_turtlesim/alica_ros_turtlesim.rosinstall 
wstool update`

Reply:
Yes, but that doesn't quite work. We can assume the 3 git repos are available. Currently, travis fails over this file because it can't install different branches of the same repo